src/dispatch/feedback/service/models.py

```diff
@@ -37,20 +39,18 @@ class ServiceFeedback(TimeStampMixin, FeedbackMixin, Base):
         )
     )
 
-    @hybrid_property
-    def project(self):
-        return self.individual.project
```

Comment:
Did this not work and that's why you had to add a project column?

Reply:
(discussed offline)